src/dealWizard/wizardManager.ts

```diff
@@ -0,0 +1,87 @@
+import { autoinject } from "aurelia-framework";
+import { PLATFORM } from "aurelia-pal";
+import { RouteConfig } from "aurelia-router";
+import { WizardService, IWizardState, IWizardStage } from "services/WizardService";
+import { DealRegistrationData, IDealRegistrationData } from "entities/DealRegistrationData";
+import { IStageMeta, WizardType } from "./dealWizardTypes";
+
+@autoinject
+export class WizardManager {
+  public wizardState: IWizardState<IDealRegistrationData>;
+  public stageMeta: IStageMeta;
+  public view: string;
+  public viewModel: string;
+  private stages: IWizardStage[] = [{
+    name: "Proposal",
+    valid: false,
+    route: "stage1",
+    moduleId: PLATFORM.moduleName("./stages/proposalStage/proposalStage"),
+  }, {
+    name: "Lead Details",
+    valid: false,
+    route: "stage2",
+    moduleId: PLATFORM.moduleName("./openProposalWizard/openProposalProposalLeadStage/openProposalProposalLeadStage"),
+  }, {
+    name: "Primary DAO",
+    valid: false,
+    route: "stage3",
+    moduleId: PLATFORM.moduleName("./stages/primaryDaoStage/primaryDaoStage"),
+  }];
+  private registrationData = new DealRegistrationData();
+
+  constructor(public wizardService: WizardService) {}
+
+  activate(params: any, routeConfig: RouteConfig): void {
```

Comment:
Could you add a type here? sth like

`StageRouterParam = { stageRoute : string }`.

Maybe also a comment, that `*stageRoute` is used in `app.ts`.
Though I'm not sure, if we should hard code file names here.
Maybe "used in wizard router configuration"